realm/realm-library/src/testObjectServer/kotlin/io/realm/log/obfuscator/LoginInfoObfuscatorTest.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.log.obfuscator
+
+import io.realm.mongodb.PatternObfuscatorFactory
+import org.junit.Test
+import kotlin.test.assertEquals
+
+const val IRRELEVANT_INPUT = """{"blahblahblah":"blehblehbleh"}"""
+
+class LoginInfoObfuscatorTest {
```

Comment:
You seem to test all the relevant things, but I'm a bit concerned there are no proper integration tests. The reason being that if ObjectStore at some point changes which payload is sent to the server, we will not detect this and these obfuscators will fail to do the right thing.

We don't need to test all the error cases, but IMO we should test the happy path towards a real server.